commit c4e676f196560500d41ff8652d6be0c735758001
Author: Anton Blanchard <anton@samba.org>
Date:   Wed Mar 25 13:43:18 2015 +1100

    powerpc: Add slbfee. instruction
    
    opcodes/ChangeLog:
    2015-03-25  Anton Blanchard  <anton@samba.org>
    
            * ppc-opc.c (powerpc_opcodes): Add slbfee.

### a/opcodes/ChangeLog
### b/opcodes/ChangeLog
## -1,3 +1,7 @@
+2015-03-25  Anton Blanchard  <anton@samba.org>
+
+	* ppc-opc.c (powerpc_opcodes): Add slbfee.
+
 2015-03-24  Terry Guo  <terry.guo@arm.com>
 
 	* arm-dis.c (opcode32): Updated to use new arm feature struct.
--- a/opcodes/ppc-opc.c
+++ b/opcodes/ppc-opc.c
@@ -5795,6 +5795,8 @@ const struct powerpc_opcode powerpc_opcodes[] = {
 {"tlbwelo",	XTLB(31,978,1),	XTLB_MASK,   PPC403,	PPCNONE,	{RT, RA}},
 {"tlbwe",	X(31,978),	X_MASK, PPC403|BOOKE|PPCA2|PPC476, PPCNONE, {RSO, RAOPT, SHO}},
 
+{"slbfee.",	XRC(31,979,1),	XRA_MASK,    POWER6,	PPCNONE,	{RT, RB}},
+
 {"stbcix",	X(31,981),	X_MASK,      POWER6,	PPCNONE,	{RS, RA0, RB}},
 
 {"icbi",	X(31,982),	XRT_MASK,    PPC|PPCVLE, PPCNONE,	{RA0, RB}},
